### PR TITLE
Allow redshift to be nullified in source page component

### PR DESCRIPTION
**Pull request by acrellin**
nullify_redshift

Closes https://github.com/fritz-marshal/fritz-beta-feedback/issues/89

### CODE REVIEW

**Comment by stefanv** [comment]
Ari, would it be hard to have a "clear" button next to SAVE instead of having the users specify "null"?  It feels like we're exposing the underbelly of the beast a bit too much :)

**Comment by dmitryduev** [comment]
I agree with @stefanv, that'll make it more user-friendly.

**Comment by acrellin** [comment]
@stefanv @dmitryduev agreed, much better (and simpler solution). Done: 
nullify_redshift

Ready for review.

**Comment by stefanv** [comment]
@dmitryduev Wording? Maybe "Clear" or "Reset"?

**Comment by acrellin** [comment]
@stefanv those sound like they apply to the form field. Nullify OTOH means to set equal to null

**Comment by acrellin** [comment]
@stefanv  Leaving the wording as is. Ready for review.

**Comment by acrellin** [comment]
How about "Set null" or "Set to null"?

**Comment by acrellin** [comment]
Updated to "Set null": 
nullify_redshift

File: static/js/components/UpdateSourceRedshift.jsx [hunk 1]
```diff
@@ -77,7 +78,7 @@ const UpdateSourceRedshift = ({ source }) => {
         }}
         style={{ position: "fixed" }}
       >
-        <DialogTitle>Update Redshift</DialogTitle>
+        <DialogTitle>Update Redshift (or Nullify)</DialogTitle>
```
Comment (guynir42) [inline static/js/components/UpdateSourceRedshift.jsx:81]: Have we decided on "null" as opposed to "clear" the redshift? 
If not, then the title of the dialog should change as well. 
If "null" then it is ok.

**Review by guynir42 — APPROVED** [review]
If we are happy with the nomenclature it all looks good. 
I think "clear" or "nullify" are both good options.

**Comment by stefanv** [comment]
I'd like to hear @dmitryduev's take on naming; I suspect users don't know what `null` means and it will confuse them.  Clear is very straightforward.

**Comment by acrellin** [comment]
It's hard to imagine any astronomer not knowing what null means. @dmitryduev can you weigh in here?

**Comment by stefanv** [comment]
:man_shrugging: I only know astronomers that program :)

**Comment by acrellin** [comment]
> 🤷‍♂️ I only know astronomers that program :)

😂 Same here, actually

**Comment by dmitryduev** [comment]
Folks, how about leaving it as is + adding a tooltip (?) on the right with a brief explanation of what it actually does?

**Comment by acrellin** [comment]
@dmitryduev @stefanv done: 
nullify_redshift_tooltip

**Review by dmitryduev — APPROVED** [review]
👍🏻

**Comment by stefanv** [comment]
Last thing I'd ask is to remove the "(or Nullify)" from the widget title.

Still leaning to "Clear" instead of "Set null", which feels unnecessarily technical to me, but I will merge either way once that is done.

Thanks, Ari, and sorry for dragging this out.

**Comment by acrellin** [comment]
@stefanv   
image
😉

**Comment by stefanv** [comment]
Now I feel a bit bad—didn't mean to strongarm that one.  Sorry :(

**Comment by stefanv** [comment]
Redemption via merging.

**Comment by acrellin** [comment]
@stefanv no worries, I'm very happy with your proposed wording 🙂

**Comment by acrellin** [comment]
> Redemption via merging.

😂

**Pull request merged**